0000-master-topology.md

```diff
@@ -0,0 +1,78 @@
+- Feature Name: (fill with a unique identifier, ex: my-awesome-feature)
+- Start Date: (fill with today's date, YYYY-MM-DD)
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+There is a well proven want and need of Salt users to be able to group minions
+into a single master logically and/or physically. Add the ablity
+to link master's together and target different masters.
+
+# Motivation
+[motivation]: #motivation
+
+There is a well proven want and need of Salt users to be able to group minions
+into a single master logically and/or physically. Historically Synic has been
+used to achive these kinds of groupings. The intention of this SEP is to create
+a more robust solutions which will eventually deprecate Synics by building this
+functionality directly into Salt Masters.
+
+# Design
+[design]: #detailed-design
+
+Add the ability to define a topology of masters. Each node under topology is a
+master. Users will be able to target jobs based on which master the minion is
+connected to in addition to our existing targeting options.
+
+```
+topology:
+  - west:
+    - west-dc-1:
+      - rack1
+      - rack2
+      - rack3
+    - west-dc-2:
+      - rack1
+      - rack3
+      - rack3
+    - west-dc-2:
+      - rack1
+      - rack3
+      - rack3
+  - east:
+    - east-dc-1:
+      - rack1
+      - rack2
+      - rack3
+    - east-dc-2:
+      - rack1
+      - rack3
+      - rack3
+```
+
+
+## Alternatives
+[alternatives]: #alternatives
+
+What other designs have been considered? What is the impact of not doing this?
```

Comment:
The targeting problem can be trivially solved with minion ids that reflect the topology.

What specifically would this proposal add?